dotnet/src/SemanticKernel.Abstractions/Planning/PlanningException.cs

```diff
@@ -6,6 +6,7 @@
 namespace Microsoft.SemanticKernel.Planning;
 
 #pragma warning disable CA1032 // Implement standard exception constructors
+#pragma warning disable RCS1194 // Implement exception constructors
```

Comment:
Do we need both rules? Can we set `RCS1194 ` to none instead?

Reply:
they don't catch all the occurrences as far as I know, we can check separately -- we won't be creating more exceptions so it's a one-off that we can clean up later